ee/clickhouse/queries/funnels/funnel_unordered.py

```diff
@@ -107,7 +107,8 @@ def _get_step_times(self, max_steps: int):
 
         for i in range(1, max_steps):
             conditions.append(
-                f"if(isNotNull(conversion_times[{i+1}]), dateDiff('second', conversion_times[{i}], conversion_times[{i+1}]), NULL) step_{i}_conversion_time"
+                f"if(isNotNull(conversion_times[{i+1}]) AND conversion_times[{i+1}] <= conversion_times[{i}] + INTERVAL {self._filter.funnel_window_interval} {self._filter.funnel_window_interval_unit_ch()}, "
```

Comment:
it's worth using `toInterval` methods here instead - less interpolation the better.

See `get_interval_func_ch`

Reply:
Agreed. There were a couple other places using the old way, and I didn't want to include refactoring all of that as part of this PR, hence didn't do this change.